spec/v3/services/requests/create_spec.rb

```diff
@@ -250,7 +262,7 @@
         "@type",
         "repository",
         "remaining_requests",
-        "2",
+        "1",
```

Comment:
This is due to the test setup, mostly. We now create the request early, so when the test setup creates 10 request records, it's now increased by 1. I believe the request throttling should now be more accurate than before.